toolchain/selects.bzl

```diff
@@ -9,18 +9,18 @@ def platform_llvm_binary(binary):
 
 def platform_extra_binary(binary):
     return select({
-        "@toolchains_llvm_bootstrapped//platforms/config:macos_aarch64": "@static-extras-toolchain-artifacts-darwin-arm64//:%s" % binary,
-        "@toolchains_llvm_bootstrapped//platforms/config:linux_x86_64": "@static-extras-toolchain-artifacts-linux-amd64//:%s" % binary,
-        "@toolchains_llvm_bootstrapped//platforms/config:linux_aarch64": "@static-extras-toolchain-artifacts-linux-arm64//:%s" % binary,
+        "@toolchains_llvm_bootstrapped//platforms/config:macos_aarch64": "@prebuilts-extras-toolchain-artifacts-darwin-arm64//:%s" % binary,
+        "@toolchains_llvm_bootstrapped//platforms/config:linux_x86_64": "@prebuilts-extras-toolchain-artifacts-linux-amd64//:%s" % binary,
+        "@toolchains_llvm_bootstrapped//platforms/config:linux_aarch64": "@prebuilts-extras-toolchain-artifacts-linux-arm64//:%s" % binary,
     })
 
 def platform_cc_tool_map(exec_os, exec_cpu):
     if exec_os == "macos":
         tool_repo = "@llvm-toolchain-minimal-%s-darwin-arm64//" % LLVM_VERSION
     elif exec_cpu == "x86_64":
-        tool_repo = "@llvm-toolchain-minimal-%s-linux-amd64//" % LLVM_VERSION
+        tool_repo = "@llvm-toolchain-minimal-%s-linux-amd64//" % (LLVM_VERSION)
```

Comment:
undo these for consistency?

Reply:
Next PR brings this, sorry, I reverted just to split windows from upgrade in case it failed.